docs/pages/guides/schema-validation.mdx

```diff
@@ -48,10 +48,26 @@ the storage that do not match the schema you provided. Situations like this
 should only happen in development mode, and the developer is responsible for
 fixing them.
 
-## Create a schema
+## Schema Operations
+
+You can interact with the schemas you create through the dashboard or by calling
+the [rest API](/docs/api-reference/rest-api-endpoints). In this guide, we will
+cover the following operations:
+
+- [Create a schema](#Create-a-schema)
+- [Delete a schema](#Delete-a-schema)
+- [Attach a schema to a room](#Attach-a-schema-to-a-room)
+- [Detach a schema from a room](#Detach-a-schema-from-a-room)
+
+You can also use the rest API to
+[list schemas](/docs/api-reference/rest-api-endpoints#list-schemas),
+[get a schema](/docs/api-reference/rest-api-endpoints#get-schema), and
+[update a schema](/docs/api-reference/rest-api-endpoints#update-schema).
+
+### Create a schema
 
 Creating a schema via the dashboard is straightforward. Simply navigate to the
-“Schemas” page in the project you want to add the schema to, and click on the
+“Schemas” page in the project you want to add the schema, and click on the
```

Comment:
Heads up: latest thinking is that we’ll remove “Schemas” from the top level nav and eventually make it more of a secondary citizen in the dashboard (accessible via the rooms list/detail pages). Steven and Pierre have some ideas/designs for that.

For now though, absolutely true. Just a reminder that we should update these instructions when we change the UI.

Reply:
thanks for the heads up :)